stable/velero/templates/deployment.yaml

```diff
@@ -90,11 +90,13 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: metadata.namespace
-          {{- if and .Values.credentials.useSecret (or (eq $provider "aws") (eq $provider "gcp")) }}
+          {{- if and .Values.credentials.useSecret (or (eq $provider "aws") (eq $provider "gcp") (eq $provider "azure")) }}
             {{- if eq $provider "aws" }}
             - name: AWS_SHARED_CREDENTIALS_FILE
-            {{- else }}
+            {{- else if eq $provider "gcp" }}
             - name: GOOGLE_APPLICATION_CREDENTIALS
+            {{- else }}
```

Comment:
please leave the else as it is and check for azure instead.

Reply:
In other words, move the azure env variable inside of a specific if statement for this provider

Reply:
azure is now checked with else if and gcp kept in else as before